File: seed/csharp-model/oauth-client-credentials-environment-variables/src/SeedOauthClientCredentialsEnvironmentVariables/Auth/TokenResponse.cs
using System.Text.Json.Serialization;

namespace SeedOauthClientCredentialsEnvironmentVariables;

public class TokenResponse
{
    [JsonPropertyName("access_token")]
    public string AccessToken { get; init; }

    [JsonPropertyName("expires_in")]
    public int ExpiresIn { get; init; }

    [JsonPropertyName("refresh_token")]
    public string? RefreshToken { get; init; }
}

File: seed/swift-sdk/oauth-client-credentials-environment-variables/src/OauthClientCredentialsEnvironmentVariables/Auth/TokenResponse.swift
public struct TokenResponse: Codable, Hashable {
    public let accessToken: String
    public let expiresIn: Int
    public let refreshToken: String?

    enum CodingKeys: String, CodingKey {
        case accessToken = "access_token"
        case expiresIn = "expires_in"
        case refreshToken = "refresh_token"
    }
}